ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -56,6 +56,7 @@
       with_items:
         - /srv/prometheus/data
         - /srv/prometheus/rules
+        - /srv/alertmanager
```

Comment:
We should create /srv/alertmanager/data – Alertmanager has its own data directory
The same should be done in pmm-update